client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx

```diff
@@ -7,13 +7,14 @@ import {
 	until,
 } from '@guardian/source-foundations';
 import {
+	Button,
 	LinkButton,
 	Stack,
 	SvgCalendar,
 	SvgClock,
 	SvgCreditCard,
 } from '@guardian/source-react-components';
-import { Link, Navigate, useLocation } from 'react-router-dom';
+import { Navigate, useLocation, useNavigate } from 'react-router-dom';
```

Comment:
Would be nice to refactor `Navigate` to `useNavigate` to keep it consistent throughout.

Reply:
yeah fair point!